eslint-config-react/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     'react/no-set-state': 1,
 
     // overridden import rules
-    'import/prefer-default-export': 2,
+    'import/prefer-default-export': 0,
     'import/extensions': 0,
```

Comment:
Like in #11 can you add `'import/named': 2` as well? This will require named exports.